### ISSUE TITLE
Blocked loading mixed active content “http://exam105_backend:port/dashboard/de/search/date”

### ISSUE THREAD

**Issue by RacketyWater7** [exam105/frontend-production#1]
### The description about mixed content as stated on developer.mozilla.org is as follows: ###
An HTTPS page that includes content fetched using cleartext HTTP is called a mixed content page. Pages like this are only partially encrypted, leaving the unencrypted content accessible to sniffers and man-in-the-middle attackers. That leaves the pages unsafe. 

This error implies that we are making data request from the client which is on an https origin, while the requested data is on an http origin. Meaning mixed. So this error is caused because of that.

**Comment by RacketyWater7** [comment]
To solve this issue, we added https with the backend url to prevent the content being labelled as mixed. But then we got another error, as below:
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://{productionBackendUrl}/dashboard/de/search/date. (Reason: CORS request did not succeed).
And the description about this error on mozilla docs is as: Trying to access an https resource that has an invalid certificate will cause this error.
While the certificate is a valid one, to be clear. So we have changed the backend url back to http.
